### PR TITLE
Document Addon Resizer configuration

### PR DESCRIPTION
**Pull request by kawych**
Preview available at https://github.com/kawych/autoscaler/tree/addon-resizer-release-1.8/addon-resizer#addon-resizer-configuration

### CODE REVIEW

**Comment by kawych** [comment]
@mwielgus @MaciekPytel PTAL (Please wait for LGTM from other reviewers before merging)

@thkoch2001 @roberthbailey - Please review this PR

@x13n - FYI

File: addon-resizer/README.md [hunk 1]
```diff
@@ -102,3 +102,146 @@ spec:
...
             name: nannyconfig
 ```
+
+## Addon resizer configuration
+
+To follow instructions in this section, set environment variable `ADDON_NAME` to
+name of your addon, for example:
+
+```
+ADDON_NAME=$ADDON_NAME
```
Comment (thkoch2001) [inline addon-resizer/README.md]: This line does not seem to make sense, you might wanted to write:
ADDON_NAME=heapster

Please add a line at the beginning of this README listing all the places that you know where this addon-resizer is in use, e.g.
In GKE the addon-resizer is used to scale heapster and ...

File: addon-resizer/README.md [hunk 2]
```diff
@@ -102,3 +102,146 @@ spec:
...
+configuration is passed to Addon Resizer via command-line flags and consists of
+parameters:
+
+1. CPU parameters:
+  ```
+  --cpu
+  --extra-cpu
+  ```
+
+  On n-node cluster, the total CPU assigned to the addon will be computed as:
```
Comment (x13n) [inline addon-resizer/README.md]: It will be at least this, possibly more, since there are buckets for multiple node counts. Same for memory below.

**Comment by kawych** [comment]
@thkoch2001 @x13n 
Thanks for the review. The comments applied in other PR https://github.com/kubernetes/autoscaler/pull/696 (I believe this readme should be actually available on master).

**Pull request closed without merging**